src/Build.UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -1712,8 +1713,10 @@ public void RegistryPropertyInvalidPrefixError2()
             }
            );
         }
-#if FEATURE_WIN32_REGISTRY
+
         [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SupportedOSPlatform("windows")]
```

Comment:
What's the difference between these?

Reply:
The former disables the test on non-Windows platforms (via Arcade magic). The latter silences the warnings about using Windows-only APIs. As far as I could tell from the API docs, there's not a way to tell the OS-checking analyzer about another attribute that has the same meaning as `SupportedOSPlatform`.